cpp/.clangd

```diff
@@ -0,0 +1,65 @@
+# https://clangd.llvm.org/config
```

Comment:
Does this imply that you've actually gotten RAFT compiling w/ clang?

Reply:
No, this file is for the clang language server daemon to provide C++ and CUDA intellisense.